vignettes/build-lifecycle.Rmd

```diff
@@ -56,7 +60,7 @@ after_script: R -q -e 'tic::after_script()'
 
 In a nutshell, the workflow is as follows:
 
-`.travis.yml`/`appveyor.yml` -> `tic.R` -> commands/step to execute
+`.travis.yml`/`appveyor.yml` -> `tic.R` -> commands/steps to execute
 
 An important point to note: The R code declared in `tic.R` is not meant to be run manually.
```

Comment:
We now can run the code manually for troubleshooting purposes, in addition to `dsl_load()` . Can you add a section describing this workflow?

Reply:
You mean an example of `run_all_stages()`?